utils/cloneByDate.py

```diff
@@ -19,16 +19,21 @@
 
 import os
 import sys
+import six
```

Comment:
I think this should be like:

```python
from salt.ext import six
```

That way we would reuse the `six` implementation we also provide in Salt. Like in https://github.com/uyuni-project/uyuni/pull/285/commits/c331136658a8c2100f5ff47e0f44127d26555784

Reply:
@meaksh ooops. This is not salt. This should not depend on salt.
Also your linked commit might need to be changed

Reply:
we decided to use `salt.ext.six` since this is available on SUSE Manager machine